package.json

```diff
@@ -83,7 +83,7 @@
   "dependencies": {
     "prop-types": "^15.5.10",
     "rc-slider": "^8.5.0",
-    "styled-components": "^2.1.1",
+    "styled-components": "^3.2.4",
```

Comment:
I *think* we should go ahead and move this to `peerDependencies` and `devDependencies` as well as allowing a looser version range (`>=2.0 || >=3.0`) to ensure this doesn't break for anyone still using v2 of styled-components

Reply:
The loose version is already there in v1.  I would love to get this to go since there are no breaking changes that we are using in v3.

Reply:
Didn't realize that it was already in peerDependencies, but this should move to devDependencies for sure